java/base-client/src/main/java/no/bankaxept/epayment/client/base/http/HttpStatus.java

```diff
@@ -1,20 +1,38 @@
 package no.bankaxept.epayment.client.base.http;
 
+import no.bankaxept.epayment.client.base.Response;
+
 import java.util.Objects;
 
 public class HttpStatus {
 
     private final int status;
 
     public HttpStatus(int status) {
-        if(status < 99 || status > 1000) throw new IllegalArgumentException("Unknown http status");
+        if (status < 99 || status > 1000) throw new IllegalArgumentException("Unknown http status");
         this.status = status;
     }
 
     public boolean is2xxOk() {
         return startsWith(2);
     }
 
+    public Response toResponse() {
+        switch (status) {
+            case 201:
+                return Response.Accepted;
+            case 422:
+                return Response.Rejected;
+            case 409:
+                return Response.Conflicted;
+            default:
+                if (is2xxOk()) return Response.Repeated;
```

Comment:
Too wide. Only 200 indicates repeat. 2xx which is not 200 nor 201 should map to Failed.

Reply:
changed